project_5.html

```diff
@@ -0,0 +1,697 @@
+			</p>
+            <p class="mb-5">
+                Small checkpoints will prepare you for the final task: You will create an MP3 player with a "Play/Pause" toggle button and a "Skip/Next Song" button,
+                using the DFPlayer Mini module. Songs are -loaded onto a microSD card which you will insert into the module. The system will output mono sound through
+                a small speaker.
+            </p>
+			<h2 class="mb-3 fw-bold" id="pj-heading-concepts">Concepts</h2>
+            <h3 class="mb-3" id="pj-heading-uart">UART</h3>
+			<p class="mb-3">
+				Embedded within the Arduino Nano is a device called the <strong>Universal Asynchronous Receiver-Transmitter (UART)</strong>. It may communicate with another UART by
+                transmitting or receiving messages known as <strong>packets</strong>. This is done in <strong>serial</strong>, meaning packets are sent one bit of data at a time. UARTs also communicate
+                <strong>asynchronously</strong>, so there is no clock to time when packets are transmitted or received.
+			</p>
+			<div style= "text-align: center;" class="mb-3">
+				<img class="img-fluid my-3 me-5"
+					src="./assets/images/project_5/ops_project_5_uart_explanation.png" width = "750">
+			</div>
+
+			<h4 class="mb-3 fw-light">Serial Communication with Arduino</h4>
+			<p class="mb-3">
+				We can use the UART in our Arduino Nano to transmit data to another UART device. To do so, we will enlist the help of the
+				<a class="link-primary"
+						href="https://www.arduino.cc/reference/en/language/functions/communication/serial/" target="_blank"
+						rel="noopener noreferrer">Serial Library</a>, which is already installed in the Arduino IDE.
+			</p>
+			<p class="mb-3">
+				The data within each packet sent with UART is <strong>one byte</strong> in size, meaning if the data contained an <code class="pj-code language-arduino">int</code>, its value could only be in the range <code class="pj-code language-arduino">-128 to 127</code> (signed) or <code class="pj-code language-arduino">0 to 255</code> (unsigned). We’ll see how this comes into play later.
+			</p>
+			<p class="mb-3">
+				Every UART can both transmit and receive packets, which is referred to as <strong>duplex communication</strong>. They communicate over their <strong>TX</strong> and <strong>RX</strong> pins; recall that the TX pin on one device connects to the RX pin on the other.
+			</p>	
+			<p class="mb-3">
+				Remember how we set the baud rate of our Nano using <code class="pj-code language-arduino">Serial.begin(9600)</code>? When a UART receives a transmission, it reads each bit at the speed of the <strong>baud rate</strong> (measured in bits per second). Both the transmitting and receiving UART <strong>must have the same baud rate</strong> for proper communication.
+			</p>
+			<p class="mb-3">
+				There are three core functions of the Serial library:
```

Comment:
This section turned out to be harder to read than I thought. (Lot of text with not a lot of spacing). Let's try creating an ordered bootstrap list out of the three paragraphs below. Each of the "core functions" should be a list item.